src/nni_manager/common/manager.ts

```diff
@@ -99,6 +99,7 @@ abstract class Manager {
     public abstract stopExperiment(): Promise<void>;
     public abstract getExperimentProfile(): Promise<ExperimentProfile>;
     public abstract updateExperimentProfile(experimentProfile: ExperimentProfile, updateType: ProfileUpdateType): Promise<void>;
+    public abstract feedTunerData(data: string): Promise<void>;
```

Comment:
This function is not specifically for tuner, we can come up with a better name, such as feedTuningData, or feedDataForTuning

Reply:
modify it to feedTuningData~